src/core/util/colorconversion.cpp

```diff
@@ -88,27 +87,19 @@ std::string rgba2hex(const vec4& rgba) {
     // change byte order
     std::swap(color.r, color.a);
     std::swap(color.g, color.b);
-
-    std::ostringstream ss;
-    ss << "#" << std::setw(8) << std::setfill('0') << std::hex
-       << *reinterpret_cast<unsigned int*>(&color);
-    return ss.str();
+    return fmt::format("#{:08x}", *reinterpret_cast<unsigned int*>(&color));
```

Comment:
any reason not to do fmt::format("#{:02x}{:02x}{:02x}{:02x}", color.a. color.b, color.g, color.r) instead?